lighthouse-core/config/full-config.js

```diff
@@ -9,25 +9,6 @@
 const fullConfig = {
```

Comment:
delete the file maybe?  breaking change opportunity and all that.

Reply:
that ended up being somewhat large of a change, so new PR https://github.com/GoogleChrome/lighthouse/pull/9930 👍